components/server/ee/src/license-source.ts

```diff
@@ -17,6 +18,13 @@ export class DBLicenseKeySource implements LicenseKeySource {
 
     async getKey(): Promise<{ key: string; domain: string; }> {
         let key: string = "";
+
+        if (this.config.licenseFile) {
```

Comment:
We need another way to inject this - it adds sync-io to sth that callers would not expect.

[Here](https://github.com/gitpod-io/gitpod/blob/7be34c57f1a190779ae923cdd11aee8e8e2a5a56/components/server/src/config.ts#L169) is the place where we load config-derived values.

Reply:
@geropl ok, that's an easy enough change. I'll do it now. Thanks

Reply:
@geropl have changed as requested and confirm it keeps the same functionality